__tests__/todo-config-test.ts

```diff
@@ -160,6 +161,62 @@ describe('todo-config', () => {
     });
   });
 
+  describe('ensureTodoConfig', () => {
+    it('does not change package.json if lintTodo directory present', () => {
+      ensureTodoStorageDirSync(project.baseDir);
+
+      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      ensureTodoConfig(project.baseDir);
+
+      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      expect(pkg).toEqual(originalPkg);
+    });
+
+    it('does not change package.json if todo config already present', () => {
+      ensureTodoStorageDirSync(project.baseDir);
+
+      writeTodoConfig(project.baseDir, {
+        warn: 10,
+        error: 20,
+      });
+
+      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      ensureTodoConfig(project.baseDir);
+
+      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      expect(pkg).toEqual(originalPkg);
+    });
```

Comment:
Can you add another test that confirms that it doesn't add config if an empty config is found (either as `{}` or `null`)?

Something like:

```js
    it('does not change package.json if todo config is empty object', () => {
      ensureTodoStorageDirSync(project.baseDir);

      writeTodoConfig(project.baseDir, {});

      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      ensureTodoConfig(project.baseDir);

      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      expect(pkg).toEqual(originalPkg);
    });

    it('does not change package.json if todo config is null', () => {
      ensureTodoStorageDirSync(project.baseDir);

      writeTodoConfig(project.baseDir, null);

      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      ensureTodoConfig(project.baseDir);

      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      expect(pkg).toEqual(originalPkg);
    });
```

Reply:
Well I think I'll just steal yours :)

Reply:
So the second test case won't work unless I change the signature of the `writeTodoConfig` function to accept `null`. I'll just add the single case.